app/services/base_api_manager.rb

```diff
@@ -7,28 +7,28 @@ def initialize(organization = "default")
   end
 
   #Fetches all the resources for the specific entity
-  def get_entities(entity_name = "", opts={})
+  def get_entities(entity_name="", opts={}, last_synchronization_date=nil, after_last_sync=false)
```

Comment:
Just to be annoying, ruby style guide specify that you should have spaces when giving default value to arguments `opts = {}`
Also maybe you have too many default values here: put a default value only in the argument is optional. entity_name is not